include/openthread/border_routing.h

```diff
@@ -520,6 +533,62 @@ otError otBorderRoutingGetNextPeerBrEntry(otInstance                           *
  */
 uint16_t otBorderRoutingCountPeerBrs(otInstance *aInstance, uint32_t *aMinAge);
 
+/**
+ * Iterates over the Recursive DNS Server (RDNSS) address entries.
+ *
+ * Address entries are discovered by processing RDNSS option in the received Router Advertisements messages from
```

Comment:
```suggestion
 * Address entries are discovered by processing the RDNSS option within received Router Advertisement messages from
```

Also, this one says "processing" while `otBorderRoutingRdnssAddrEntry` above uses "parsing".  Should they be the same?

Those two and the one below in `otBorderRoutingRdnssAddrCallback` should probably align to the same wording

Reply:
Changed all in the new pushed commit. Thanks.